app/controllers/time_tracker_extension/telegram_controller.rb

```diff
@@ -48,6 +86,12 @@ def with_locale(&block)
       I18n.with_locale(locale, &block)
     end
 
+    def with_time_zone
+      # TODO: change this when we start to keep timezone in user's model
+      timezone = 3
```

Comment:
It's temp, but I think it's healthier  to use config.time_zone from the app.

Reply:
config.time_zone is UTC, so you would have problems now in case of launching these commands from 0 to 3 AM